openedx/features/qverse_features/registration/context_manager.py

```diff
@@ -0,0 +1,28 @@
+"""
+Contains context managers of QVerse registration app.
+"""
+from django.conf import settings
+
+
+class one_time_password_link_expiry:
```

Comment:
I don't know if it was intentional or what, but in my opinion, we should differentiate between class names and method names.

Reply:
It is intentional as normally, we use lower cases for context managers. You can take a look at Django's official code

https://github.com/django/django/blob/03537e245886ed1811444c0993025dc2e08ac38b/django/test/utils.py#L392